src/Microsoft.VisualStudio.Threading.Analyzers.Tests/Microsoft.VisualStudio.Threading.Analyzers.Tests.csproj

```diff
@@ -26,7 +26,7 @@
     <PackageReference Include="Microsoft.VisualStudio.Shell.Interop.11.0" Version="11.0.61030" IncludeAssets="none" />
     <PackageReference Include="xunit" Version="2.2.0" />
     <PackageReference Include="xunit.runner.visualstudio" Version="2.2.0" />
-    <PackageReference Include="MicroBuild.Nonshipping" Version="2.0.42" />
+    <PackageReference Include="MicroBuild.Nonshipping" Version="2.0.43" />
```

Comment:
Version of this package looks like a good candidate for being an MSBuild property in Directory.build.props.

Reply:
Good idea. Done.